pleskdistup/common/tests/rpmtests.py

```diff
@@ -232,12 +233,19 @@ def test_write_exsisted_repodata(self):
 
 
 class HandleRpmnewFilesTests(unittest.TestCase):
+    def __init__(self, methodName: str = "runTest") -> None:
+        self.test_dir = "rpm_test_dir"
```

Comment:
Better to create a class variable, since you don't intend to set it per-instance anyway.